### PR TITLE
*: upgrade etcd to v3.4.3

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve? 
upgrade `etcd` to 3.4.3 to fix https://github.com/etcd-io/etcd/pull/10772

### What is changed and how it works?
upgrade

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)

### CODE REVIEW

**Comment by overvenus** [comment]
make tidy fails.

File: pkg/metricutil/metricutil.go [hunk 1]
```diff
@@ -60,12 +61,12 @@ func camelCaseToSnakeCase(str string) string {
 
 // prometheusPushClient pushs metrics to Prometheus Pushgateway.
 func prometheusPushClient(job, addr string, interval time.Duration) {
+	pusher := push.New(addr, job)
+	pusher = pusher.Gatherer(prometheus.DefaultGatherer)
+	pusher = pusher.Grouping("instance", instanceName())
```
Comment (shafreeck) [inline pkg/metricutil/metricutil.go]: Would you like to combine these lines together to use a chain-style like:
`pusher := push.New(addr, job).Gatherer(prometheus.DefaultGatherer).Grouping(...)`

**Comment by nolouch** [comment]
PTAL @shafreeck

**Pull request merged**